communication/slack-config/channels.yaml

```diff
@@ -5,9 +5,11 @@ channels:
   - name: africa-dev
   - name: airflow-operator
   - name: akri
-  - name: aks-engine-users
+  - name: aks
+    id: CU3N85WJK
```

Comment:
Theres unfortunately already an aks user and they share a namespace :( we can't rename the channel to that.

Reply:
Hmm, in terms of discoverability, could we perhaps go with azure-aks or aks-azure? I'm hoping for it to come up in searches.